src/components/appNavigation/stackNavigation.js

```diff
@@ -255,11 +256,19 @@ class AppView extends Component<AppViewProps, AppViewState> {
     this.setState(state => ({ currentState: { ...state.currentState, ...data } }))
   }
 
+  goToExternal = () => {
+    // should use rdu url from fvContext
+    redirectTo('https://gooddapp.org?verified=false')
```

Comment:
looks mostly correct. don't forget that there are 2 states. CBU and RDU look how we use them.
CBU = popup mode, REST call to url, close window
RDU = redirect mode, redirect to url